GitUI/BranchTreePanel/RepoObjectsTree.Nodes.cs

```diff
@@ -165,51 +174,45 @@ protected void ReloadNodes(Func<CancellationToken, Task> loadNodesTask)
 
                 ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
                 {
+                    Nodes.Clear();
+
+                    var token = _reloadCancellationTokenSequence.Next();
```

Comment:
`_reloadCancellationTokenSequence` needs to be Disposed when left panel is.

Reply:
> `_reloadCancellationTokenSequence` needs to be Disposed when left panel is.

Yes, you're right. I will make sure to dispose of these when the left panel is disposed.